CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## Unreleased
 
+- [#827](https://github.com/XenitAB/terraform-modules/pull/827) VMSS source_image_reference provider bug workaround.
```

Comment:
It should be under ### Fixed. Please update

Reply:
Thanks, done